libs/core/src/lib/grid-list/components/grid-list-group-header/grid-list-group-header.component.html

```diff
@@ -1,6 +1,6 @@
 <div
     class="fd-grid-list__group-header fd-grid-list__group-header--no-margin"
-    tabindex="0"
+    tabindex="-1"
```

Comment:
Removing `tabindex` attribute at all will make this element not focusable as it's the div.